GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigAdvancedSettingsPage.Designer.cs

```diff
@@ -58,20 +59,31 @@ private void InitializeComponent()
             this.checkBoxPullRebase.AutoSize = true;
             this.checkBoxPullRebase.Location = new System.Drawing.Point(19, 14);
             this.checkBoxPullRebase.Name = "checkBoxPullRebase";
-            this.checkBoxPullRebase.Size = new System.Drawing.Size(276, 17);
+            this.checkBoxPullRebase.Size = new System.Drawing.Size(303, 19);
             this.checkBoxPullRebase.TabIndex = 1;
             this.checkBoxPullRebase.Text = "Rebase local branch when pulling (instead of merge)";
             this.checkBoxPullRebase.UseVisualStyleBackColor = true;
             // 
+            // checkBoxRebaseAutosquash
+            // 
+            this.checkBoxRebaseAutosquash.AutoSize = true;
+            this.checkBoxRebaseAutosquash.Location = new System.Drawing.Point(19, 116);
+            this.checkBoxRebaseAutosquash.Name = "checkBoxRebaseAutosquash";
+            this.checkBoxRebaseAutosquash.Size = new System.Drawing.Size(367, 19);
+            this.checkBoxRebaseAutosquash.TabIndex = 3;
+            this.checkBoxRebaseAutosquash.Text = "Automatically squash commits when doing an interactive rebase";
+            this.checkBoxRebaseAutosquash.UseVisualStyleBackColor = true;
+            // 
             // GitConfigAdvancedSettingsPage
             // 
             this.AutoScaleDimensions = new System.Drawing.SizeF(96F, 96F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Dpi;
+            this.Controls.Add(this.checkBoxRebaseAutosquash);
```

Comment:
nit pick: keep options in the same order as they are used

Reply:
(If I understand well) done.